app/views/layouts/application.html.erb

```diff
@@ -12,8 +12,11 @@
   <body>
   <%= render "layouts/header" %> 
   <div class="container">
+    <% flash.each do |message_type, message| %>
+      <div class="alert alert-<%= message_type %>"><%= message %></div> 
+    <% end %>
     <%= yield %> 
   </div>
 	
   </body>
-</html>
+</html>
```

Comment:
Please, add an empty line

Reply:
done

Reply:
You have not yet fixed this mistake, add an empty line after row 23

Reply:
done